src/topology/algebra/order/basic.lean

```diff
@@ -2071,6 +2071,46 @@ begin
     { exact h.2.symm ▸ tendsto_const_mul_zpow_at_top_zero h.1} }
 end
 
+@[priority 100] -- see Note [lower instance priority]
+instance linear_ordered_field.to_has_continuous_inv₀ : has_continuous_inv₀ α :=
```

Comment:
It's tempting to directly prove the division_ring result. Does this work without further modifications?
```suggestion
@[priority 100] -- see Note [lower instance priority]
instance linear_ordered_field.to_topological_division_ring : topological_division_ring α :=
```